Sources/ToucanSDK/ContextStore/ContextStore.swift

```diff
@@ -171,14 +172,13 @@ struct ContextStore {
             )
             do {
                 contents = try executor.execute()
+                didRenderHTML = true
             }
             catch {
                 logger.error("\(String(describing: error))")
             }
         }
-
-        let didRenderHTML = pipelines.map { $0.render }.contains(true)
-
+        
         if didRenderHTML {
```

Comment:
```suggestion
        if pipeline.render {
```

Reply:
I believe this is incorrect and was the source of our confusion as well. After running a pipeline, we expect HTML content because:
	•	We have already converted the markdown to HTML in the script (i.e., render=false), or
	•	We want to use the built-in converter (i.e., render=true).

In either case, the outcome of the pipeline should be HTML content.

Reply:
It would be much more beneficial to use this flag to indicate whether the script converts the content to HTML or not. This would overcome the limitations of the current approach and enable the possibility of running multiple pipelines.
`rendersHTML=true` (default: false)